src/Commands.pm

```diff
@@ -4025,9 +4025,9 @@ sub cmdPortalList {
 		#Command: portals add y_airport 143 43 y_airport 148 51 0 c r0 c r0
 		print $args."TEST\n";
 		my ($srcMap, $srcX, $srcY, $dstMap, $dstX, $dstY, $seq) = $args =~ /^add ([a-zA-Z\_\-0-9]*) (\d{1,3}) (\d{1,3}) ([a-zA-Z\_\-0-9]*) (\d{1,3}) (\d{1,3})(.*)$/; #CHECKING
-		my $srcfile = $srcMap.'.fld';
+		my $srcfile = $srcMap.'.fld.gz';
```

Comment:
Should write compatibility code so the old `.fld` will still work. See line 3292 above for an example.